apps/explorer/components/Home/index.tsx

```diff
@@ -150,13 +147,13 @@ export function Home({
                   color="contrast"
                 >
                   {getDownloadCost({
-                    price: metrics?.average.settings.download_price,
+                    price: metrics?.priceTable.downloadbandwidthcost,
                     exchange,
                   })}
                 </Text>
                 <Text color="subtle">
                   {getDownloadCost({
-                    price: metrics?.average.settings.download_price,
+                    price: metrics?.priceTable.downloadbandwidthcost,
```

Comment:
Not related to this PR but question: Are all these metrics fields averages? Do we plan to support things besides average now or in the future? If so should we put these under an averages key?

Reply:
They are averages, yes. @chris124567 may have an answer here. I assume you mean like min and max?